docs/creating_integration_manifest.md

```diff
@@ -83,6 +83,10 @@ Requirements are Python libraries or modules that you would normally install usi
 
 Requirements is an array of strings. Each entry is a `pip` compatible string. For example, the media player Cast platform depends on the Python package PyChromecast v3.2.0: `["pychromecast==3.2.0"]`.
 
+:::tip
+The `__init.py__` of your integration must be able to load without any requirements.
```

Comment:
This isn't true. Requirements will be installed before loading the integration modules.